generate/get-version.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+script_dir="$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)"
+find $script_dir -type f | xargs cat | shasum | cut -d ' ' -f 1
```

Comment:
Why not just `find "$(dirname "${BASH_SOURCE[0]}")" ....`?

Reply:
Why are you shasuming the cat'd contents of all the scripts?
If that's really what you wanted to do you may as well

`tar cp "$script_dir" | shasum | cut -d\  -f 1`